From 97f9fa95b6eaa436d855d31ceb8f6c364d64d7bf Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Sun, 6 Aug 2017 01:35:25 +0200
Subject: [PATCH 171/206] Snap: Don't close slide out menu after selecting
 scene mode

Keep consistent with how filter mode menu behaves.

Change-Id: If53bdca075bf97e1f2ad3b8a3f494dcb720026dc
---
 src/com/android/camera/PhotoMenu.java | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/com/android/camera/PhotoMenu.java b/src/com/android/camera/PhotoMenu.java
index 460acc3df..6a892224c 100644
--- a/src/com/android/camera/PhotoMenu.java
+++ b/src/com/android/camera/PhotoMenu.java
@@ -1154,7 +1154,6 @@ public class PhotoMenu extends MenuController
                             for (View v1 : views) {
                                 v1.setActivated(v1 == v);
                             }
-                            animateSlideOutPreviewMenu();
                         }
 
                     }
-- 
2.17.1

